src/Surfnet/ServiceProviderDashboard/Domain/Entity/ManageEntity.php

```diff
@@ -18,6 +18,8 @@
 
 namespace Surfnet\ServiceProviderDashboard\Domain\Entity;
 
+use SebastianBergmann\Type\UnknownType;
+use Surfnet\ServiceProviderDashboard\Application\Metadata\JsonGenerator;
```

Comment:
Rule of thumb, no application or infra dependencies should be present in the domain layer. This also looks like a dev-leftover as I see no further references to JsonGenerator.